### ISSUE TITLE
Feature: godep remove

### ISSUE THREAD

**Issue by ansel1** [tools/godep#325]
It would be a nice feature to support `godep remove` which could selectively remove a vendored dependency.  Sometimes it's nice to temporarily remove a dependency, to test some uncommitted code changes in the GOPATH version of a package.

This is especially true in our project, where we have a number of independent packages, which all vendor and common shared lib.  We are switching the use GO15VENDOREXPERIMENT, and noticed it's no longer very convenient to concurrently make changes to the shared libs and the other packages that depend on it.

**Comment by freeformz** [comment]
Agreed. It's pretty low on *my* priority list atm, given the other issues related to `vendor/` vs. Godeps workspace vs. Godeps workspace with rewrites. If you would like to take a crack at it though that would be great.

**Comment by freeformz** [comment]
Closing issues before archiving. Thank You\!
